.k8s/app.values.yml

```diff
@@ -0,0 +1,46 @@
+# Default values for Mas webapp. https://github.com/SocialGouv/helm-charts/blob/master/charts/webapp/values.yaml
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# See https://github.com/SocialGouv/helm-charts/tree/master/charts/app
+---
+image:
+  repository: $CI_REGISTRY_IMAGE
+  tag: master
+
+# nameOverride: app
+
+deployment:
+  resources:
+    requests:
+      cpu: 100m
```

Comment:
```suggestion
      cpu: 5m
```

is enough no ?

Reply:
I thought we should have request / limit quit the same ?